apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/IterableLikeToContainInAnyOrderOnlyCreatorSamples.kt

```diff
@@ -0,0 +1,136 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.verbs.expect
+import kotlin.test.Test
+import ch.tutteli.atrium.api.infix.en_GB.*
+
+class IterableLikeToContainInAnyOrderOnlyCreatorSamples {
+    @Test
+    fun value() {
+        expect(listOf("A")) toContain o inAny order but only value("A")
```

Comment:
almost, you can also remove the last parentheses:
```suggestion
        expect(listOf("A")) toContain o inAny order but only value "A"
```
Same same below for other `value` cases, you need the parentheses for `values` though

Reply:
ok Rob, I made the changes